README.md

```diff
@@ -1192,6 +1192,15 @@ This setting is [auto updated](#gmkdxsettingsauto_updateenable) when available.
 let g:mkdx#settings = { 'enter': { 'enable': 1 } }
 ```
 
+## `g:mkdx#settings.enter.increment`
+
+This setting defines increment done for each new item in numbered list, Default value is `1` setting `0` will keep current index on new item.
+
+```viml
+" :h mkdx-setting-enter-increment
```

Comment:
I see that a mention to `doc/mkdx.txt` is made here via helptag, but no section for this has been created, can you add the relevant section in the table of contents of `doc/mkdx.txt` and then also put the section itself explaining what it does in that file as well?

Reply:
I have updated the help docs.

Reply:
Alright, we're almost there :) As you can see on Github this README.md has a table of contents almost all the way at the top, this item is not included there, the same is true for the doc/mkdx.txt file. Though since the docs are there I'll add the TOC entries where needed. Cheers for the quick addition :+1: